example/06-volumesnapshotlocation.yaml

```diff
@@ -24,6 +24,6 @@ spec:
     bucket: <YOUR_BUCKET>
     prefix: <PREFIX_FOR_BACKUP_NAME>
     backupPathPrefix: <PREFIX_FOR_BACKUP_PATH>
-    provider: <GCP_OR_AWS>
+    provider: <gcp_OR_aws>
```

Comment:
If there are any know gotchas around permissions or how the buckets are created, please add them as comments in this YAML.

Reply:
ok